### PR TITLE
feat: #4902 pipeline tear down 2

### PR DESCRIPTION
**Pull request by bashleigh**
# Pull request checklist

**Detail as per issue below (required):**

fixes: #4902

### CODE REVIEW

File: packages/deployment-service/src/services/task.ts [hunk 1]
```diff
@@ -68,13 +68,12 @@ export const deleteTasksFromPipeline = async (pipeline: PipelineEntity): Promise
...
+  const tasks = await repo
     .createQueryBuilder('t')
-    .leftJoin('t.pipelineRunner', 'pr', 'pr.id = t.pipeline_runner_id')
-    .leftJoin('pr.pipeline', 'p', 'p.id = pr.pipeline_id')
-    .where('p.id = :pipelineId', {
-      pipelineId: pipeline.id,
-    })
-    .delete()
-    .execute()
+    .leftJoin('t.pipelineRunner', 'pr')
```
Comment (willmcvay) [inline packages/deployment-service/src/services/task.ts:73]: The Yorkshire pipeline runner...

**Review by willmcvay — APPROVED** [review]
LGTM 👍🏻

File: packages/deployment-service/src/constants.ts [hunk 2]
```diff
@@ -9,4 +9,5 @@ export enum QueueNames {
   CODE_BUILD_VERSION_DEPLOY = 'https://sqs.eu-west-2.amazonaws.com/028446965111/CodebuildDeploy',
   PIPELINE_SETUP = 'https://sqs.eu-west-2.amazonaws.com/028446965111/PipelineSetup',
   PIPELINE_TEAR_DOWN = 'https://sqs.eu-west-2.amazonaws.com/028446965111/PipelineTearDown',
+  PIPELINE_TEAR_DOWN_START = 'https://sqs.eu-west-2.amazonaws.com/028446965111/PipelineTearDownStart',
```
[the remark this answers is not in the document]
Comment (bashleigh) [reply packages/deployment-service/src/constants.ts:12]: yeaaa I hate this bit. It needs sorting. Problem was that with enums you can't use calculated values like `https://sqs...${process.env.PIPELINE_QUEUE}` so I need to refactor this at some point!

File: packages/deployment-service/src/services/task.ts (hunk 1 above)
Comment (bashleigh) [reply packages/deployment-service/src/services/task.ts:73]: 🍵

File: packages/deployment-service/src/constants.ts (hunk 2 above)
[the remark this answers is not in the document]
Comment (joshbalfour) [reply packages/deployment-service/src/constants.ts:12]: ahhh kk, npnp!

**Pull request merged**